### PR TITLE
Only `portable` builds (binaries)

### PR DESCRIPTION
**Pull request by antondlr**
See #5489 for details, this is a cleaner version of the release/binaries part of that PR

For backwards compatibility, we still publish a copy of the binaries with the `portable`name. This is meant to be phased out as we direct our users towards the regular (now, portable) release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -166,6 +131,7 @@ jobs:
                   tar -czf lighthouse-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.arch }}.tar.gz lighthouse.exe
                   gpg --passphrase "$env:GPG_PASSPHRASE" --batch --pinentry-mode loopback -ab lighthouse-${{ needs.extract-version.outputs.VERSION }}-${{ matrix.arch }}.tar.gz
                   move *tar.gz* ..
+                  # todo copy to `-portable`
```
Comment (michaelsproul) [inline .github/workflows/release.yml]: I think this comment can be deleted (already done above)

**Review by michaelsproul — APPROVED** [review]
approved modulo removing that comment

File: .github/workflows/release.yml (hunk 1 above)
Comment (michaelsproul) [inline .github/workflows/release.yml]:
```suggestion
```

**Comment by michaelsproul** [comment]
@mergifyio queue

**Comment by realbigsean** [comment]
@mergify queue

**Comment by realbigsean** [comment]
@mergify queue

**Comment by realbigsean** [comment]
@mergify requeue

**Pull request merged**